docs/shared/recipes/tips-n-tricks/switch-to-workspaces-project-references.md

```diff
@@ -234,8 +289,10 @@ The project's `tsconfig.spec.json` does not need to reference project dependenci
 
 ```jsonc {% fileName="libs/ui/tsconfig.spec.json" %}
 {
-  "extends": "./tsconfig.json",
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
+    // outDir should be local to the project and not in the same folder as any other tsconfig.*.json
+    "outDir": "./out-tsc/spec"
```

Comment:
why? Is it safe to have everything (build outputs) in the workspace root outdir? Any reasons to not do it that way?